alpha-order-string/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Alphabetical ordering",
+  "name": "alpha-order-string",
```

Comment:
In the meta.json file, besides "name:" you put a function name different to the function name in the starter code.